include/itkWASMPolyDataToPolyDataFilter.hxx

```diff
@@ -189,8 +189,8 @@ WASMPolyDataToPolyDataFilter<TPolyData>
     const rapidjson::Value & pointsJson = document["points"];
     const std::string pointsString( pointsJson.GetString() );
     CoordRepType * pointsPtr = reinterpret_cast< CoordRepType * >( std::atol(pointsString.substr(35).c_str()) );
-    polyData->GetPoints()->resize(numberOfPoints);
-    polyData->GetPoints()->assign(pointsPtr, pointsPtr + numberOfPoints);
+    polyData->GetPoints()->resize(numberOfPoints * 3);
+    polyData->GetPoints()->assign(pointsPtr, pointsPtr + numberOfPoints * 3);
```

Comment:
@thewtex Do you mean something like:

    CoordRepType * coordPtr = reinterpret_cast<CoordRepType *>(std::atoll(pointsString.substr(35).c_str()));

    for (PointType & point : *(polyData->GetPoints()))
    {
      for (auto & coord : point)
      {
        coord = *coordPtr;
        ++coordPtr;
      }
    }

I'm sorry, but I think it cannot be written as a single `assign` call, because the right side consists of individual coordinate values, while the left side consists of points.

Reply:
The correct changes is the one suggested by @Leengit below.